lib/test-util-macros/src/lib.rs

```diff
@@ -0,0 +1,143 @@
+// The MIT License (MIT)
```

Comment:
Is there a way we could just include their macro? I'm concerned we'll drift from the upstream if we copy it like this.

Reply:
Do you mean copy their code and only add `crate::test_util::trace_init()`?

Reply:
I was more just wondering if we could embed their macro somehow, but I don't know if that is possible. I just didn't want to miss updates to the upstream macro.